src/module/api.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-unused-vars */
+/* eslint import/no-cycle: 0 */
+
+import SHOWS from './home.js';
+
+const baseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
+const appsEndPoint = '/apps/';
+const likesEndPoint = '/likes/';
+const id = 'DJQ0xNDyZupm6FxK1BuA';
+
+const popUpOverlay = document.querySelector('.popup-overlay');
+
+export default class INTERACTIONS {
```

Comment:
- [ ] By JavaScript naming standards, your class names should be named using `PascalCase`. It is similar to Camel Case except that the first letter of the word must be Capitalized.
Therefore your `INTERACTIONS` class should be named `Interactions`.

✔️ Please make this change everywhere you have named the classes using the wrong naming convention.

**NB:** Please don't forget to update your default exports and imports due to this change in order to avoid errors 👍